link.sh

```diff
@@ -42,7 +41,7 @@ for exe in ./*; do
 			name="$link"
 		fi
 
-		ln -fs "$exe" "./$name"
+		ln -fs "$base" "$bindir/$name"
 	fi
```

Comment:
All of thise changes look good. I will merge them and test them in my local copy and be sure they work on my machine.